tests/templates/kuttl/delete-rolegroup/00-install-zk.yaml.j2

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+metadata:
+  name: install-kafka-zk
+timeout: 300
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: kafka-zk
+spec:
+  servers:
+    roleGroups:
+      default:
+        replicas: 1
+        config:
+          myidOffset: 10
+  version: {{ test_scenario['values']['zookeeper-latest'] }}
+  stopped: false
```

Comment:
The config and myidOffset can be removed if we use only one rolegroup. Stopped can be removed as well.

Reply:
Done.